packages/gateway-client/src/worker/index.ts

```diff
@@ -620,7 +642,7 @@ async function main() {
         connectionManager: {
             autoDial: true, // Auto connect to discovered peers (limited by ConnectionManager minConnections)
             minConnections: 3,
-            maxDialsPerPeer: 10,
+            maxDialsPerPeer: 20,
```

Comment:
Is this limiting the number of connections? Or the number of retries for a given peer?

Reply:
it's how many multiaddrs to try in parallel.